### ISSUE TITLE
List of repositories providing wine-pba

### ISSUE THREAD

**Issue by Espionage724** [acomminos/wine-pba#15]
I'm not too sure if anyone else is maintaining a repo with Wine and these patches this soon, but I figure it might be convenient to list them here.

I maintain a Fedora Copr here: https://copr.fedorainfracloud.org/coprs/espionage724/wine-patches-pba/

**Comment by stefson** [comment]
I can push an ebuild for gentoo to my overlay, but only for >=wine-staging-3.0 - please ping me if interested.

**Comment by thrimbor** [comment]
I uploaded an Arch Linux AUR package here: https://aur.archlinux.org/packages/wine-staging-pba/
Don't know how frequently I can update that though.

**Comment by lexming** [comment]
In Gentoo it is easy to try the wine-pba patches without a specific ebuild/repo by placing all the patch files inside /etc/portage/patches/app-emulation/wine-staging-2.21. Portage will automatically apply the wine-pba patches on top of the official wine-staging-2.21 every time it is emerged.

**Comment by TomBZombie** [comment]
@thrimbor, thanks for helping all your fellow, but lazy, arch users out. :)

**Comment by Saroufim** [comment]
Alternatively you can all download it through Lutris.

**Comment by asharin75** [comment]
Yep, I download the Lutris build to use with PlayonLinux lol  https://lutris.net/files/runners/

**Comment by IngeniousDox** [comment]
Lutris build has been updated to: https://github.com/acomminos/wine-pba/commit/68de8e9b3f26e68bc6d64f353e0954ddab2f7590

I think the goal is to keep that as the "base" wine-pba build in lutris, since it works great and smooth. Any new builds will prolly be called wine-pba-test (or something). This allows users to test newer versions, but also have a base version to fall back on should there be a regression/bug.

**Comment by acomminos** [comment]
Feel free to use [this wiki page](https://github.com/acomminos/wine-pba/wiki/Repositories) to post repos.

**Comment by FireBurn** [comment]
Gentoo: FireBurn Overlay

I've added pba as a use flag  to the wine-any-3.3.ebuild in my overlay

**Comment by IngeniousDox** [comment]
Lutris: Staging 3.3 + PBA now available (Including AMD patch that is confirmed to work for WoW fps)

Staging 2.21 will be updated to the latest commit for Staging when the build queue is empty at some point.

**Comment by IngeniousDox** [comment]
Updated Lutris installers for Blizzard Games to use Wine-PBA where applicable. More info: [On Reddit](https://www.reddit.com/r/wine_gaming/comments/82pvcd/lutris_blizzard_games_installers_updated_to_wine/)

**Comment by IngeniousDox** [comment]
PS: We cannot change the wiki. Tried to add the Lutris info there aswell.

Was going to add, we have: pba-2.21 and pba-3.3 builds.

**Comment by acomminos** [comment]
> We cannot change the wiki.

Should be fixed now.

**Comment by mrdeathjr28** [comment]
If any one interested, this is my wine 3.3 vanilla i386 + pba + donat enikeev csmt patch in deb package

https://mega.nz/#!nY0wgQQC!iRnIeVxWMmpFj0vCIiWnEsRqf3zzaIa5mwOZaJ9do7c

:)

**Comment by goldpaw** [comment]
I've added an AUR version for arch linux that uses the latest git versions of wine, wine-staging and wine-pba as opposed to the stable/tagged releases we already have a package for.

I've also added this to the wiki of this project: 
https://aur.archlinux.org/packages/wine-staging-pba-git/

**Comment by mrdeathjr28** [comment]
wine 3.4 + pba + donat enikeev csmt patch + svesop novsync patch (32bit apps only)

https://mega.nz/#!eU0jXQ5R!t6cXfXeM-kgVaepe3-qw69__A39L6Jw5M4FV6hfpx7M

**Comment by mati865** [comment]
Added https://build.opensuse.org/package/show/home%3Aahjolinna/wine%3Astaging-pba to the wiki for openSUSE (not mine repo).
Posting here as well.

**Comment by dreamhunt** [comment]
Add wine-pba 3.10 staging to Lutris or elsewhere please :)

**Comment by IngeniousDox** [comment]
If someone is willing to maintain a wine-staging patched with wine-pba with the ENV vars from Firerat. Then it is just a matter of poking Strider, he can then throw it into his buildbot and you will have pba versions for each version of Wine. Right now he has to manually add the patches each time, and he doesn't want to spend time on that.

**Comment by mati865** [comment]
@IngeniousDox is there something wrong with Firerat repo: https://github.com/Firerat/wine-staging?

**Comment by IngeniousDox** [comment]
Those are just the patches, not wine patched. Strider will make wine builds if you hand him a patched version of Wine (So he doesn't have to figure out how or what to patch himself). So someone would need to take those patches, apply them to wine, push them to a git repo, hand it to Strider. And update that every 2 weeks for a new version of Wine.

**Comment by ghost** [comment]
well, that is easy
I have cron running ( a slightly newer version ) of [this script](
https://gist.github.com/Firerat/bb4a2e795dbb8aabe9a90fa92d97b635)

All you would need to do is add build scripts/trigger to the TagPBA function
That function checks if wine-staging has a new tag, if so it tags and pushes changes. fully automated \o/

I guess I could just fork wine-patched. I'm more likely to do that if/when wine-staging moves to gitlab ( which is easy ! )

btw, at the moment my scripts are pushing to both github and gitlab, at some point I will drop github.
 
Edit:
a little note on the script, it has many `if fail exit` it doesn't report the actual problem. Thus far it has been fine, i.e. no merge conflicts. I'm not sure I can script fixing a merge conflict.. in the least I should write a proper exit function to report nature of problem, or where it occured )

**Comment by ghost** [comment]
done
https://gitlab.com/Firer4t/wine-pba-patched/

**Comment by IngeniousDox** [comment]
Aight, thx Firerat. Passed the URL on to Strider. "pba-firerat-3.10" is now available in Lutris.

**Comment by camdenorrb** [comment]
3.13? :3

**Comment by IngeniousDox** [comment]
esync-3.13 includes pba.

**Comment by dotiful** [comment]
@thrimbor @goldpaw  unfortunately no more available(
both 404 error

**Comment by MEXAHOTABOP** [comment]
pathset is outdated 
today you can use dxvk with even better perfomance

**Comment by qwertychouskie** [comment]
DXVK isn't for all systems though, e.g. Ivy Bridge and Haswell have very bare-bones Vulkan support.

**Comment by mirh** [comment]
Well, if you can narrow down what's the issue with them, you can ask mesa/intel devs to fix that (if they have some spare time). 

Said this, anyway, sooner or later wine should be getting a something along this patchset. 
https://www.winehq.org/pipermail/wine-devel/2019-June/147386.html
And maybe people will stop to think it's the API doing 90% of the magic.
